doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -63,27 +63,15 @@ To render the documentation on your own machine:
 
 0. Ensure that you have a working SciPy :ref:`dev-env` active.
    You need to be able to ``import scipy`` regardless of Python's working
-   directory; the ``python setup.py develop`` and ``conda develop`` commands
-   from the :ref:`quickstart <dev-env>` guides make this possible.
-#. Install `Sphinx`_, `PyData Sphinx theme`_, `Sphinx-Design`_
+   directory; see :ref:`quickstart <dev-env>` and our other guides.
+#. Install `Sphinx`_, `PyData Sphinx theme`_, `Sphinx-Design`_, `numpydoc`_
    and `matplotlib`_. For example, if you're using the Anaconda distribution of
    Python, enter in a terminal window::
-      
-      conda install sphinx pydata-sphinx-theme sphinx-design matplotlib --channel conda-forge
+
+      conda install sphinx pydata-sphinx-theme sphinx-design matplotlib numpydoc --channel conda-forge
       
    The list of requirements is in ``scipy/doc_requirements.txt``.
-#. In a terminal window, browse to the ``scipy/doc`` directory. Note the
-   presence of the file ``Makefile``.
-#. Execute ``git submodule update --init``.
-   Some of the documentation theme files are not distributed
-   with the main ``scipy`` repository; this keeps them up to date using
-   `git submodules`_.
```

Comment:
We still need to keep this, though, right?

Reply:
I removed that since it's covered by the step 0. I think we should try as much as possible to not repeat ourselves.

Reply:
Oh I see. Thanks for clarifying, that makes sense